Dockerfile

```diff
@@ -1,10 +1,12 @@
 ARG PLATFORM=""
-FROM hmctspublic.azurecr.io/base/node${PLATFORM}:12-alpine as base
+FROM hmctspublic.azurecr.io/base/node${PLATFORM}:16-alpine as base
 
 USER root
 RUN apk add --no-cache python3 py3-pip make gcc g++
 COPY package.json yarn.lock ./
-RUN yarn install --production
+RUN corepack enable
+RUN yarn plugin import workspace-tools
```

Comment:
could this be added into the `yarnrc` file so we don't need to install in docker each time?